2024-12-21 19:16:04,687 - asyncio - DEBUG - Using proactor: IocpProactor
2024-12-21 19:16:04,690 - root - INFO - Searching for GoPro cameras...
2024-12-21 19:16:04,814 - root - INFO - Discovered GoPro: C3461325908060._gopro-web._tcp.local. at 172.20.160.51
2024-12-21 19:16:04,859 - root - INFO - Discovered GoPro: C3461326043977._gopro-web._tcp.local. at 172.29.177.51
2024-12-21 19:16:04,888 - root - INFO - Discovered GoPro: C3461326029723._gopro-web._tcp.local. at 172.27.123.51
2024-12-21 19:16:09,695 - root - INFO - Found the following GoPro devices:
2024-12-21 19:16:09,697 - root - INFO - Name: C3461325908060._gopro-web._tcp.local., IP: 172.20.160.51
2024-12-21 19:16:09,698 - root - INFO - Name: C3461326043977._gopro-web._tcp.local., IP: 172.29.177.51
2024-12-21 19:16:09,698 - root - INFO - Name: C3461326029723._gopro-web._tcp.local., IP: 172.27.123.51
2024-12-21 19:16:09,699 - root - INFO - Using data directory: c:\!AndriiShramko\inventions\SDKGOPROShramko\GoProControl\api\v2\v39\data
2024-12-21 19:16:09,701 - root - INFO - Cache updated successfully with newly discovered devices.
2024-12-21 19:16:09,701 - root - INFO - Resetting and enabling USB control on all cameras...
2024-12-21 19:16:09,702 - root - INFO - Resetting USB control on camera 172.20.160.51.
2024-12-21 19:16:09,703 - root - INFO - Resetting USB control on camera 172.29.177.51.
2024-12-21 19:16:09,703 - root - INFO - Resetting USB control on camera 172.27.123.51.
2024-12-21 19:16:09,708 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.27.123.51:8080
2024-12-21 19:16:09,710 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.29.177.51:8080
2024-12-21 19:16:09,711 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.20.160.51:8080
2024-12-21 19:16:09,771 - urllib3.connectionpool - DEBUG - http://172.27.123.51:8080 "GET /gopro/camera/control/wired_usb?p=0 HTTP/11" 200 3
2024-12-21 19:16:09,773 - root - INFO - USB control disabled on camera 172.27.123.51.
2024-12-21 19:16:09,786 - urllib3.connectionpool - DEBUG - http://172.20.160.51:8080 "GET /gopro/camera/control/wired_usb?p=0 HTTP/11" 500 3
2024-12-21 19:16:09,788 - root - ERROR - Failed to disable USB control on camera 172.20.160.51. Status Code: 500. Response: {}

2024-12-21 19:16:09,800 - urllib3.connectionpool - DEBUG - http://172.29.177.51:8080 "GET /gopro/camera/control/wired_usb?p=0 HTTP/11" 200 3
2024-12-21 19:16:09,802 - root - INFO - USB control disabled on camera 172.29.177.51.
2024-12-21 19:16:11,777 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.27.123.51:8080
2024-12-21 19:16:11,792 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.20.160.51:8080
2024-12-21 19:16:11,807 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.29.177.51:8080
2024-12-21 19:16:11,865 - urllib3.connectionpool - DEBUG - http://172.27.123.51:8080 "GET /gopro/camera/control/wired_usb?p=1 HTTP/11" 200 3
2024-12-21 19:16:11,866 - root - INFO - USB control enabled on camera 172.27.123.51.
2024-12-21 19:16:11,880 - urllib3.connectionpool - DEBUG - http://172.20.160.51:8080 "GET /gopro/camera/control/wired_usb?p=1 HTTP/11" 200 3
2024-12-21 19:16:11,881 - root - INFO - USB control enabled on camera 172.20.160.51.
2024-12-21 19:16:11,895 - urllib3.connectionpool - DEBUG - http://172.29.177.51:8080 "GET /gopro/camera/control/wired_usb?p=1 HTTP/11" 200 3
2024-12-21 19:16:11,897 - root - INFO - USB control enabled on camera 172.29.177.51.
2024-12-21 19:16:11,897 - root - INFO - Synchronizing time on all cameras...
2024-12-21 19:16:11,899 - root - ERROR - Failed to sync time: cannot import name 'sync_time' from 'date_time_sync' (c:\!AndriiShramko\inventions\SDKGOPROShramko\GoProControl\api\v2\v39\date_time_sync.py)
2024-12-21 19:16:11,900 - root - ERROR - An error occurred: cannot import name 'sync_time' from 'date_time_sync' (c:\!AndriiShramko\inventions\SDKGOPROShramko\GoProControl\api\v2\v39\date_time_sync.py)
